DataProcessing/datax-host/src/main/scala/datax/client/eventhub/EventHubBase.scala

```diff
@@ -21,7 +24,43 @@ object EventHubBase {
       .build
   }
 
+  def extractConnectionString(connString: String) : Map[String, String] = {
+    var extracted = Map[String, String]()
+    connString.split(";").foreach(s => {
+      val parts = s.split("=")
+      if (parts.length == 2) {
+        extracted += (parts(0) -> parts(1))
+      }
+    })
+    extracted
+  }
+
+  def buildClientFromConnString(extracted: Map[String, String]): Boolean = {
+    extracted.getOrElse("SharedAccessKeyName", "") != "" &&
+      extracted.getOrElse("SharedAccessKey", "") != ""
+  }
+
   def getOutputClient(connString: String) = {
```

Comment:
With RBAC enabled for eventhub, a connection string is not a requirement. How would you handle the case when connString is not passed in